app/model/testdata/CreateEventData.scala

```diff
@@ -17,18 +17,18 @@
 package model.testdata
 
 import model.command.testdata.CreateEventRequest.CreateEventRequest
-import model.persisted.eventschedules.{ Event, EventType, VenueType }
-import org.joda.time.{ LocalDate, LocalTime }
-import play.api.libs.json.{ Json, OFormat }
+import model.persisted.eventschedules._
+import org.joda.time.{LocalDate, LocalTime}
+import play.api.libs.json.{Json, OFormat}
 import services.testdata.faker.DataFaker.Random
 
 object CreateEventData {
 
   case class CreateEventData(id: String,
```

Comment:
This object is also in faststream-admin-frontend and acceptance tests, remember to change it to match. The reason they are there is we use this class in TDG and TDG is of course called from those projects. If you forgot, it may not happen much but it may.